opcua/service.go

```diff
@@ -37,6 +43,9 @@ type Service interface {
 
 	// DisconnectThing removes thing and channel connection route-map
 	DisconnectThing(string, string) error
+
+	// Browse browses available nodes for a given OPC-UA Server URI and NodeID
+	Browse(string, string) ([]string, error)
```

Comment:
I saw the term `Discovery` in OPC-UA terminology and discovery endpoint on Server. Does it makes more sense then Browse or its not the same/related ?